jupyter-docker/jupyter_localize_extension.py

```diff
@@ -89,7 +94,11 @@ def post(self):
     if not all(map(lambda v: type(v) is unicode, pathdict.values())):
       raise HTTPError(400, "Body must be JSON object of type string/string")
 
-    async = self.get_query_argument('async', False)
```

Comment:
Unrelated, but I also noticed this bug: `get_query_argument` returns a string, so it was acting in async mode if the `async` argument was set to _anything_, including the string "false". Changed it to call `distutils.util.strtobool` to interpret it as a boolean.